roles/cluster/cluster_undeploy_kepler/tasks/main.yml

```diff
@@ -0,0 +1,15 @@
+- name: Grab kepler-operator source
+  command:
+    git clone https://github.com/sustainable-computing-io/kepler-operator "{{ artifact_extra_logs_dir }}/kepler"
```

Comment:
same as above for the destination directory

Reply:
Done!